### ISSUE TITLE
Allow mute/unmute via headset hardware buttons (plus maybe other hw button actions)

### ISSUE THREAD

**Issue by cybrjestr** [bigbluebutton/bigbluebutton#19909]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I use my Jabra Engage 75 headset which has a hardware (HW) mute/unmute button right on the headset as well as the base station, but I can mute/unmute BBB exclusively via the SW button in the browser. In MS Teams, the software (SW) is aware of the mute/unmute status I set via the hardware buttons. Synchronization between HW and SW makes it much easier to avoid "you are still muted" situations.

**Describe the solution you'd like**
When I mute/unmute a headset via a hardware button, BBB's mute status should be synchronized to the HW button.

**Describe alternatives you've considered**
No alternative, having two separate HW and SW mute statuses is confusing.

**Additional context**
I have no idea how this is implemented in detail, but I take it there are SDKs etc. that headset manufactures provide that should allow this in principle. E.g. the Jabra SDK: https://developer.jabra.com/site/global/sdk/index.gsp

**Comment by cybrjestr** [comment]
PS: Mute/unmute is the top priority here, but once that works, other buttons might be added too - eg. "quit call"/hang up.
PPS: I am just realizing that this applies to webcams, too - my webcam has an indicator to show that it is recording, which stays off however in BBB meetings.

**Comment by ffdixon** [comment]
Checking out the link, it looks like they do have a SDK for Browser.  No promises on when we can implement this, but it does look like there is a path.

**Comment by cybrjestr** [comment]
Hi, any hopes of this getting implemented? If nothing else, this comment serves to prevent auto-closing as stale.

**Comment by ffdixon** [comment]
Yes, please keep it open.  As long as there is interest in the feature, as we continue to build out the BigBlueButton platform it may become easier to implement such features.
